CHANGELOG.asciidoc

```diff
@@ -172,6 +172,10 @@ https://github.com/elastic/beats/compare/v6.1.3...v6.2.0[View commits]
 - Docker and Kubernetes modules are now GA, instead of Beta. {pull}6105[6105]
 - Support haproxy stats gathering using http (additionaly to tcp socket). {pull}5819[5819]
 - Support to optionally 'de dot' keys in http/json metricset to prevent collisions. {pull}5957[5957]
+- De dot keys in jolokia/jmx metricset to prevent collisions. {pull}5957[5957]
```

Comment:
@ph Probably that is also too much ;-)

Reply:
what where this madness is coming from, I just used the cherry pick tool with one commit o_o